scripts/mainnet/reset.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -x
+
+usage() {
+  cat << EOT
+Usage: $0 <<BUILD ID>>
+Reset fractals mainnet, example BUILD ID = e1ca4db_20211011_1204
+EOT
+}
+
+subsitute_node_build_id() {
+  ssh $1 "PERL_BADLANG=0 perl -pi -e \"s/\\\\w+_\\\\d+_\\\\d+/$2/\" docker-compose.yml"
```

Comment:
Would it be simpler to use ENV variable substitution in the docker-compose file? https://docs.docker.com/compose/environment-variables/